qiskit_metal/designs/design_base.py

```diff
@@ -338,51 +345,17 @@ def connect_pins(self, comp1_id: int, pin1_name: str, comp2_id: int,
             self._components[comp2_id].pins[pin2_name].net_id = net_id
         else:
             logger.warning(
-                f'NetId was not added for {comp1_id}, {pin1_name}, {comp2_id}, {pin2_name} and will not be added to components.'
+                f'NetId was not added for {comp1_id}, {pin1_name},'
+                f' {comp2_id}, {pin2_name} and will not be added to components.'
             )
         return net_id
 
-    # NOTE: Think nothing is using this. Remove this if no-one complains.
```

Comment:
Please keep this,  Allows user to obtain a QComponent by interger id, vs string id.

Reply:
Okay. 
Is there any issue with me deleting the comment instructing it to be deleted?

Reply:
No, just want to keep the method.. Thanks.

Reply:
The method has been commented out.  Please put it back in as active code.